int32 width
int32 height
float64 size_mm
string node_name
---
bool success
